client/my-sites/hosting/sftp-card/index.js

```diff
@@ -116,28 +117,51 @@ const SftpCard = ( {
 		);
 	};
 
+	const displayQuestionsAndButton = ! ( disabled || username || isLoading );
+
 	return (
 		<Card className="sftp-card">
 			<MaterialIcon icon="cloud" size={ 32 } />
-			<CardHeading>{ translate( 'SFTP Information' ) }</CardHeading>
+			<CardHeading>{ translate( 'SFTP Credentials' ) }</CardHeading>
 			<div className="sftp-card__body">
-				{ disabled || username || isLoading ? (
+				<p>
+					{ translate( "Access and edit your website's files directly by using an FTP client." ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'Access and edit your website's files directly using an FTP client.' )` *ES Score: 8*